_data/mentors.yml

```diff
@@ -36,3 +36,7 @@
 - name: Sanskar Jethi
   image: sanskar.jpg
   github: stealthanthrax
+
+- name: Mohit Sharma
+  image: ms.jpeg
```

Comment:
Try to keep the image format consistent with other images (Use jpg).

Reply:
ok

Reply:
doing

Reply:
Squash your commit into one at the end.

Reply:
hey some problem in my git desktop, resolving closing pr, making new one, sorry for the confusion